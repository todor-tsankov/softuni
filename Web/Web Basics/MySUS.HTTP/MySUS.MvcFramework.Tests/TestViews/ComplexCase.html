<!DOCTYPE html>

<html>
<head>
    <meta charset="utf-8" />
    <title></title>
</head>
<body>
    <h1>"Hello from toshko server {O:O}"</h1>

    @if(Model.Name == "Test1234")
    {
        <h1>This Model.Name = @Model.Name, Model.Age =@Model.Age</h1>
        <h1>@(1+1)</h1>
    }

    @for(var a = 0; a < 5; a++)
    {
        @var b = 5;
        <p>@b</p>
    }

</body>
</html>
