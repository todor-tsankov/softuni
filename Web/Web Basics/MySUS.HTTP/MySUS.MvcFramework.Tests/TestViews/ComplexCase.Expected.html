<!DOCTYPE html>

<html>
<head>
    <meta charset="utf-8" />
    <title></title>
</head>
<body>
    <h1>"Hello from toshko server {O:O}"</h1>

        <h1>This Model.Name = Test1234, Model.Age =10</h1>
        <h1>2</h1>

        <p>5</p>
        <p>5</p>
        <p>5</p>
        <p>5</p>
        <p>5</p>

</body>
</html>
